kickstarts/partials/post/source_setup.ks.erb

```diff
@@ -16,7 +16,7 @@ pushd $appliance_root
 popd
 
 mkdir -p $app_root
-git clone <%= @manageiq_checkout.remote %> $vmdb_root
+git clone --separate-git-dir /tmp/manageiq.git <%= @manageiq_checkout.remote %> $vmdb_root
```

Comment:
Yes, the next time you fetch, it will fetch everything.  But in the short term you save ~100M